src/app.js

```diff
@@ -1,11 +1,19 @@
-const path = require('path');
 const express = require('@feathersjs/express');
-const favicon = require('serve-favicon');
-const compress = require('compression');
-const cors = require('cors');
 const feathers = require('@feathersjs/feathers');
 const configuration = require('@feathersjs/configuration');
 const commons = require('@schul-cloud/commons');
+
+const app = express(feathers());
+const config = configuration();
+app.configure(config);
+
+// init & register configuration
+(new commons.Configuration()).init({ app });
+
+const path = require('path');
+const favicon = require('serve-favicon');
+const compress = require('compression');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'compression'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)